frame/society/src/lib.rs

```diff
@@ -1431,7 +1442,7 @@ impl<T: Trait<I>, I: Instance> Module<T, I> {
 						}
 					});
 
-				if approval_count < rejection_count {
+				if approval_count <= rejection_count {
```

Comment:
It makes sense for this to be skewed toward rejection. If the society member cannot even vote for themselves in their own membership challenge, they should be kicked from society.